.github/workflows/publish_deploy_image.yaml

```diff
@@ -81,20 +80,58 @@ jobs:
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
 
-  deploy-to-openshift:
-    needs: build-and-push-image
+  deploy-to-openshift-development:
+    if: github.event_name == 'pull_request'
+    needs: [build-and-push-image]
+    runs-on: ubuntu-latest
+    environment:
+      name: Development
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Deploy
+        uses: ./.github/actions
+        with:
+          openshift_server_url: ${{ env.OPENSHIFT_SERVER }}
+          openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
+          namespace: bc5936-dev
```

Comment:
I was having trouble pulling this out of the environment secrets, but I figured having it here explicitly would be fine. Any thoughts @wenzowski?